### ISSUE TITLE
a better explosive liquid

### ISSUE THREAD

**Issue by basedkit** [studio-minus/ppg-bugs-and-requests#975]
I think a new liquid similar to nitroglycerin would be pretty useful because nitroglycerin basically does nothing when detonated on its own. However, if a new liquid with better explosive properties is added it would be extremely useful for people who would want to make contraptions with liquid powered vehicles.

**Comment by vda8** [comment]
Follow the template maybe?

**Comment by basedkit** [comment]
> Follow the template maybe?

yeah ig i could mod it in myself, but an official liquid would still be pretty cool

**Comment by DindinYT37** [comment]
> > Follow the template maybe?
> 
> yeah ig i could mod it in myself, but an official liquid would still be pretty cool

That's... not what he meant.

**Comment by 404CoderNotFound** [comment]
> > Follow the template maybe?
> 
> yeah ig i could mod it in myself, but an official liquid would still be pretty cool

That's not what he said...

**Comment by basedkit** [comment]
> > > Follow the template maybe?
> > 
> > 
> > yeah ig i could mod it in myself, but an official liquid would still be pretty cool
> 
> That's not what he said...



> > > Follow the template maybe?
> > 
> > 
> > yeah ig i could mod it in myself, but an official liquid would still be pretty cool
> 
> That's... not what he meant.

yeah sorry I misunderstood, I rewrote it with the correct template
